protocol/forks/hf-20230515.md

```diff
@@ -0,0 +1,12 @@
+# HF-20230515
+
+The May 2023 hard fork is the second upgrade to be comprised of [CHIPs](#chips).
```

Comment:
where is this link supposed to go?

Reply:
Not sure, picked it up when c&p-ing the line from here haha: https://github.com/SoftwareVerde/bitcoin-cash-specification/blob/master/protocol/forks/hf-20220515.md

Reply:
...I'm also not sure where that's supposed to go.  I know that anchor links are supported, but that doesn't appear to be a valid anchor.